build/configs/rtl8720e/loadable_apps/defconfig

```diff
@@ -432,11 +438,19 @@ CONFIG_GPIO=y
 #
 # CONFIG_LCD is not set
 CONFIG_BCH=y
-# CONFIG_RTC is not set
-# CONFIG_WATCHDOG is not set
+CONFIG_RTC=y
+# CONFIG_RTC_DATETIME is not set
+# CONFIG_RTC_HIRES is not set
+# CONFIG_RTC_ALARM is not set
+CONFIG_RTC_DRIVER=y
+CONFIG_RTC_IOCTL=y
+CONFIG_WATCHDOG=y
+CONFIG_WATCHDOG_DEVPATH="/dev/watchdog0"
+# CONFIG_WATCHDOG_FOR_IRQ is not set
 # CONFIG_TIMER is not set
-# CONFIG_MMINFO is not set
+CONFIG_MMINFO=y
 # CONFIG_PRODCONFIG is not set
+CONFIG_KLOGBUFFER=y
```

Comment:
No config in public. Please confirm you check this with menuconfig.

Reply:
Done